### DISCUSSION TITLE
Aggregation Calculation Method

### DISCUSSION THREAD

**Discussion by dwi2214** [akvorado/akvorado#674] [Q&A] [answered]
Hi,
I was wondering if you could help me understand how the aggregation feature works in Akvorado. Specifically, I am curious about the calculation method used for aggregating data over time. Is it based on the average over 5 minutes within the last 6 hours?

The reason I am asking is that when I filter the data for the last hour, I noticed that some usage above 8 Mbps (for example) is displayed, which is not visible when using the last 6 hours filter. I would appreciate any insight you could provide on this matter.

Thank you

Last 6 hours
![image](https://user-images.githubusercontent.com/116805958/236365713-e984319b-e25b-4180-861f-163649b23471.png)

Last 1 hour
![image](https://user-images.githubusercontent.com/116805958/236365879-f3561120-220d-4f81-becd-00c87488189f.png)

**Comment by vincentbernat** [comment, marked as answer]
What's aggregated is the amount of bytes. The last hour is using the raw flows and is doing an average over 18 seconds. The last 6 hours is using aggregated data over one minute and is using an average over the minute. So, the peak you see on one disappear on the other because it is smoothed over.

Also see See https://github.com/akvorado/akvorado/issues/56.
